packages/cdk8s-cli/lib/util.ts

```diff
@@ -32,14 +32,25 @@ export async function withTempDir(dirname: string, closure: () => Promise<void>)
 
 async function get(url: string, protocol: typeof http | typeof https = https): Promise<string> {
   return new Promise((ok, ko) => {
-    const req = protocol.get(url, res => {
-      if (res.statusCode !== 200) {
-        throw new Error(`${res.statusMessage}: ${url}`);
+    const req = protocol.get(url, async res => {
+      switch(res.statusCode) {
+        case 200: {
```

Comment:
What about other 2xx codes?

Reply:
I want to avoid handling all statuses unless we know we'll need them. Are there any specific 2xx you want to handle?